src/__test__/graphql-flow.test.js

```diff
@@ -342,6 +342,67 @@ describe('graphql-flow generation', () => {
             `);
         });
 
+        it('should handle an inline fragment on an interface without a typeCondition', () => {
+            const result = rawQueryToFlowTypes(
+                `
+                query SomeQuery {
+                    hero(episode: JEDI) {
+                        id
+                        ... {
+                            name
+                        }
+                    }
+                }`,
+                {readOnlyArray: false},
+            );
+            expect(result).toMatchInlineSnapshot(`
+                export type SomeQueryType = {|
+                    variables: {||},
+                    response: {|
+                  hero: ?({|
+                    id: string,
+
+                    /** The person's name*/
+                    name: ?string,
+                  |} | {|
+                    id: string,
+                    name: ?string,
+                  |})
```

Comment:
This could be collapsed into a single object type.  It seems like any fragment without a type condition could be collapsed, is that correct?  Also, there's no documentation on `name` in the second element in the union type.  Is that intentional?

Reply:
Yeah, the name in the Droid definition doesn't have documentation! I think it does make sense to have it separate, as the docstrings could be different? idk, but I'm open to optimizing it in the future